worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/WorkflowWorker.java

```diff
@@ -143,4 +149,35 @@ public void processDocument(final Document document) throws DocumentWorkerTransi
             document.addFailure("WORKFLOW_SCRIPT_EXCEPTION", e.getMessage());
         }
     }
+    
+    private void addMdcLoggingData(final Document document)
```

Comment:
I'd suggest updating this to take the `Task` object rather than the `Document` object.